manifests/mod/authz_default.pp

```diff
@@ -1,3 +1,9 @@
-class apache::mod::authz_default {
-  ::apache::mod { 'authz_default': }
+class apache::mod::authz_default(
+    $apache_version = $::apache::apache_version
+) {
+  if versioncmp($apache_version, '2.4') >= 0 {
```

Comment:
I've seen this in many other modules, for example https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/mod/worker.pp#L65, and simply assumed that there must be a good reason that I have too little Puppet knowledge to understand... So what?